src/vector/platform/ElectronPlatform.tsx

```diff
@@ -377,10 +377,12 @@ export default class ElectronPlatform extends VectorBasePlatform {
         return this.ipc.call("getAvailableSpellCheckLanguages");
     }
 
-    public getSSOCallbackUrl(fragmentAfterLogin?: string): URL {
-        const url = super.getSSOCallbackUrl(fragmentAfterLogin);
+    public getSSOCallbackUrl(forOidc = false, fragmentAfterLogin?: string): URL {
```

Comment:
this is a public method, document new params please. What effect does setting this have in practice?

Reply:
they're documented on the superclass, no point duplicating content where it can drift I would have thought

Reply:
ok, please link to the superclass then

```suggestion
    /** Implementation of {@link BasePlatform.getSSOCallbackUrl}. */
    public getSSOCallbackUrl(forOidc = false, fragmentAfterLogin?: string): URL {
```